Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,31 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
-        }
+            // Loop twice. Can be changed by altering the number that "i" is less than.
```

Comment:
This is what a sample comment would look like in a PR. All looks good :)

Reply:
Thanks!